langchain/utilities/duckduckgo_search.py

```diff
@@ -0,0 +1,62 @@
+"""Util that calls DuckDuckGo Search.
+
+No setup required. Free.
+https://pypi.org/project/duckduckgo-search/
+"""
+from typing import Dict, List
+
+from duckduckgo_search import ddg
```

Comment:
We'll want to import this lazily during validation and in the appropriate functions since it shouldn'tb e a strict dependency of the project